src/algebra/module/basic.lean

```diff
@@ -11,7 +11,7 @@ import group_theory.group_action.group
 # Modules over a ring
 
 In this file we define
-
+*
```

Comment:
?

Reply:
Whoops, meant to add `distrib_mul_action_with_zero` here before deciding it would go in `algebra.smul_with_zero`